low-level/sensors/impl/platforms/realstor/realstor_disk_sensor.py

```diff
@@ -146,11 +147,15 @@ def run(self):
             # insertion/removal detected
             self._rss_check_disks_presence()
 
-            # Polling system status
-            self.rssencl.get_system_status()
+            #Do not proceed further if latest disks info is not valid due to consul connection error
+            if not self.invalidate_latest_disks_info:
+                # Polling system status
+                self.rssencl.get_system_status()
 
-            # check for disk faults & raise if found
-            self._rss_check_disk_faults()
+                # check for disk faults & raise if found
+                self._rss_check_disk_faults()
+            else:
+                logger.warn("Ignore disk faults check due to consul error")
```

Comment:
same comment as line 150

Comment can be more to the point
"Can not validate disk faults or presence due to persistence store error"

Reply:
done